locales/en.json

```diff
@@ -1683,7 +1684,7 @@
 			"archive": "Archive",
 			"calendar": "Calendar",
 			"contact": "Contact",
-			"courseData": "My course files",
```

Comment:
why is this change here? Doesnt seem to be related to the ticket, and I am not sure its actually correct...

Reply:
the course data still there i place something before then i remove the line maybe is just change the line but the data still there